server/config/conditions.ts

```diff
@@ -0,0 +1,38 @@
+export default {
+  standardConditions: [
+    {
+      text: 'Be of good behaviour and not behave in a way which undermines the purpose of the licence period.',
+    },
```

Comment:
Do standard conditions need a sequence? Or rely on the order they appear in the list?

Reply:
At the moment, conditions will appear in the order they are in config. Actually at the moment, standard conditons are not used anywhere just yet

Reply:
API will accept them on create call :-)